dotcom-rendering/src/web/components/MatchNav.tsx

```diff
@@ -300,6 +233,8 @@ export const MatchNav = ({
 			</Row>
 			{comments && <Comments comments={comments} />}
 		</StretchBackground>
-		{minByMinUrl && <MatchTabs minByMinUrl={minByMinUrl} />}
+		{minByMinUrl && (
+			<MatchTabs minByMinUrl={minByMinUrl} matchUrl={matchURL} />
+		)}
```

Comment:
Shall we move the rendering of `MatchTabs` into the layout files? That way `MatchNav` can purely be about rendering the match nav stuff and doesn't need to be concerned with tab stuff